plugins/scaffolder/src/components/TemplatePage/TemplatePage.tsx

```diff
@@ -141,11 +136,11 @@ export const TemplatePage = ({
   );
 
   const handleCreate = async () => {
-    const id = await scaffolderApi.scaffold(
+    const { taskId } = await scaffolderApi.scaffold({
       templateName,
```

Comment:
yeah again, double check that this now is handled well since the ref parsing needs full refs